twirp/errors.py

```diff
@@ -20,6 +20,10 @@ class Errors(Enum):
     DataLoss = "data_loss"
     NoError = ""
 
+    @classmethod
+    def has_value(cls, value):
+        return value in cls._value2member_map_
```

Comment:
Risky API, can we save a set instead on first run and then keep using that?

Reply:
why it is risky? is it because it is internal API? 

other alternative would be handling it in try/catch block

Reply:
ya try catch sounds good